### ISSUE TITLE
Two select events fired when opening calendar

### ISSUE THREAD

**Issue by Niek** [Wikiki/bulma-calendar#210]
Running version 6.0.7. Here's my code:

```html
<input type="date" class="input" id="date" name="date">
```

```js
bulmaCalendar.attach('#date', {
  dateFormat: "DD-MMM-YYYY",
  type: 'date',
  showClearButton: false
});
document.querySelector('#date').bulmaCalendar.on('select', date => {console.log(date)});
```
Whenever I click on the calendar, I get 2 "select" events, and another one when actually selecting a date. Is there any fix or workaround to ignore the first 2 select events?

**Comment by Niek** [comment]
Codepen with the issue:  https://codepen.io/NiekvdMaas/pen/LYYxJYL

It seems to appear only if you have pre-selected a date, either with a ``value=yyyy-mm-dd`` in the date field or from a previous selection.

**Comment by Niek** [comment]
I found a workaround: compare ``date.data.value()`` with the current date in the event callback. It's still impossible to distinguish current date selection from the automatically fired events though.

**Comment by michael-hack** [comment]
Should be fixed with 6.1.2
